<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Message</class>
 <widget class="QDialog" name="Message">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
	color: rgb(255, 255, 255);
	background-color: rgb(51, 51, 51);
	border-radius: 15px;
}
QTextBrowser
{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border-radius: 15px;
}
/*
QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on { 
    padding-top: 3px;
    padding-left: 4px;
}


QComboBox::down-arrow:on { 
    top: 1px;
    left: 1px;
}*/
QTableWidget
{
	background-color: rgb(51, 51, 51);
	alternate-background-color: rgb(113, 112, 114);
	color: rgb(255, 255, 255);
	selection-background-color: rgb(170, 0, 255);
	selection-color: rgb(255, 255, 255);
	gridline-color: rgb(255, 255, 255);
	font: 9pt &quot;Century Gothic&quot;;
}
QTableWidget QTableCornerButton::section
{
	background-color: rgb(51, 51, 51);
	color: rgb(255, 255, 255);
}
QLineEdit
{
	background: transparent;
	border: none;
	font: 12pt &quot;Century Gothic&quot;;
	border-bottom: 1px solid #717072;
	color: rgb(255, 255, 255);
}
QPushButton
{
	font: 700 10pt &quot;Century Gothic&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(170, 0, 255);
	border-radius: 10px;
}
QPushButton:Hover
{
	color: rgb(255, 255, 255);
	background-color: rgb(148, 0, 222);
}
QPushButton#removeButton
{
	font: 700 10pt &quot;Century Gothic&quot;;
	border: 1px solid rgb(255, 0, 0);
	color: rgb(255, 255, 255);
	background-color: rgb(51, 51, 51);
}
QPushButton#removeButton:Hover
{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 0, 0);
}
QMessageBox
{
	background-color: rgb(51, 51, 51);
}
QMessageBox QLabel
{
	border: 0;
	border-radius: 6px;
	font-size: 10px;
	font-weight: bold;
	padding-left: 5px;
	padding-right: 5px;
	padding-top: 5px;
	padding-bottom: 5px;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>751</width>
     <height>461</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_receivedMessages">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>481</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_message">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>390</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Write Message Here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sendAttachment">
    <property name="geometry">
     <rect>
      <x>628</x>
      <y>380</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Attach</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combobox_receivers">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_contactadd">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Contact</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sendMessage">
    <property name="geometry">
     <rect>
      <x>628</x>
      <y>410</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
